src/parser.js

```diff
@@ -0,0 +1,40 @@
+const bent = require('bent');
+const jsdom = require('jsdom');
+
+const request = bent('string');
+const url = 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List';
+const { JSDOM } = jsdom;
+const nameTest = /^name/i;
+const linkTest = /^#/;
+let lines = '';
+let feed = '';
+let feedList = '';
+let tempLine = '';
+
+/**
+* Get feed data from url
+* process it to remove unnecessary information like "name" and square brackets
+*/
+request(url)
+  .then((data) => {
+    const dom = new JSDOM(data);
+    feedList = dom.window.document.querySelector('pre').textContent;
+    lines = feedList.split(/\r\n|\r|\n/);
+
+    lines.forEach((element) => {
+      if (!linkTest.test(element)) {
+        if (element.startsWith('[')) {
+          tempLine = element.replace(/\[href="|\[/, '');
```

Comment:
I wonder if we should just change the wiki to fix that one `[href=...` example you found, so it matches the correct format?

Reply:
Also, special cases like this deserve a comment above to explain why you're bothering with this extra work to replace here.

Reply:
I removed href from wiki and added some more comments to functions